spring-cloud-aws-autoconfigure/src/test/java/io/awspring/cloud/autoconfigure/config/s3/S3ConfigDataLoaderIntegrationTests.java

```diff
@@ -22,16 +22,14 @@
 
 import io.awspring.cloud.autoconfigure.AwsSyncClientCustomizer;
 import io.awspring.cloud.autoconfigure.ConfiguredAwsClient;
+import io.awspring.cloud.autoconfigure.imds.ImdsAutoConfiguration;
 import io.awspring.cloud.autoconfigure.s3.S3ClientCustomizer;
 import java.io.IOException;
 import java.time.Duration;
 import java.util.Map;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
-import org.springframework.boot.BootstrapRegistry;
-import org.springframework.boot.BootstrapRegistryInitializer;
-import org.springframework.boot.SpringApplication;
-import org.springframework.boot.WebApplicationType;
+import org.springframework.boot.*;
```

Comment:
I will be that guy :)

Reply:
I imagine spotless should have taken care of that